commands/publish_dafsa.py

```diff
@@ -0,0 +1,100 @@
+import requests
+from requests.exceptions import HTTPError
+import tempfile
+import subprocess
+from kinto_http import Client, KintoException
+import sys
+
+COMMIT_HASH_URL = "https://api.github.com/repos/publicsuffix/list/commits/master"
+
+LIST_URL = (
+    "https://raw.githubusercontent.com/publicsuffix/list/master/public_suffix_list.dat"
+)
+
+MAKE_DAFSA_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/make_dafsa.py"
+PREPARE_TLDS_PY = "https://raw.githubusercontent.com/arpit73/temp_dafsa_testing_repo/master/publishing/prepare_tlds.py"
+
+
+SERVER = "https://kinto.dev.mozaws.net/v1"
+CREDENTIALS = ("arpit73", "s3cr3t")  # (username, password)
+
+BUCKET_ID = "firefox-core-network-dns"
+COLLECTION_ID = "public-suffix-list"
+RECORD_ID = "latest-commit-hash"
+
+
+def handle_request_errors(func):
+    def inner(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except HTTPError as http_err:
+            print(f"HTTP error occurred: {http_err}")
+        except Exception as err:
+            print(f"Other error occurred: {err}")
+        else:
+            print("Fetching Failed")
```

Comment:
Why is it better to print errors than to let them crash the program?  We may loose context/traceback/etc. no?